game/hud/src/widgets/Map/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * @Author: Justin Vick (justin.zezmin@gmail.com)
+ * @Date: 2017-09-08 16:32
+ * @Last Modified by: Justin Vick (justin.zezmin@gmail.com)
+ * @Last Modified time: 2017-09-08 16:32
+ */
+
+import * as React from 'react';
+import MapMain from './components/MapMain';
+
+export interface MapContainerProps {
+  containerClass?: string;
+  visibleComponent: string;
+}
+
+export interface MapContainerState {
+  visible: boolean;
+  mapLoaded: boolean;
+}
+
+const map = new Image();
```

Comment:
this should be a member of the class to make it more reusable.  You can initialize it in the constructor.